apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/PublisherServiceApiClient.cs

```diff
@@ -1909,6 +1940,71 @@ public virtual Publisher.PublisherClient GrpcClient
             throw new sys::NotImplementedException();
         }
 
+        /// <summary>
+        /// Lists the names of the snapshots on this topic.&lt;br&gt;&lt;br&gt;
+        /// &lt;b&gt;ALPHA:&lt;/b&gt; This feature is part of an alpha release. This API might be
```

Comment:
This is in alpha server-side.
So I assume it shouldn't be in a GA client library?
Why is this not in a pubsub-alpha service?

Reply:
Yup, I find that alarming too. For now I'll remove the whole pubsub commit, and we can investigate separately.